### DISCUSSION TITLE
WDACConfig - custom base policy

### DISCUSSION THREAD

**Discussion by Kofl** [HotCakeX/Harden-Windows-Security#324] [General]
Hi,

we have a custom base policy and one supplemental policy, which mainly whitelists %ProgramFiles% and %windir%)

Should we be able to create an additional supplemental policy with WDACConfig, if our WDAC policy is already applied on the system?
(currently running in a Powershell DataTime Error, and dont want to open a bug if we use it as not designed :-) )

If that is not supported, how would we build an additional supplemental policy, when we can set only the baseline policy in WDACConfig and no additional policies for creating an supplemental policy

Thanks in advance

**Comment by HotCakeX** [comment]
Hi, of course, check out the documentations here, you can do so many things with the module
https://github.com/HotCakeX/Harden-Windows-Security/wiki/WDACConfig

You need the XML file of your base policy to create supplemental policies. Some cmdlets allow you to only use the base policy's GUID for supplemental policy creation, such as [ConvertTo-WDACPolicy](https://github.com/HotCakeX/Harden-Windows-Security/wiki/ConvertTo-WDACPolicy).

About the error, i'm not sure what it is you're referring to, need more info.

**Reply by Kofl** [reply]
Awesome, thanks a lot. We will run the module again and open a issue if it should re-occur.

**Reply by HotCakeX** [reply]
Anytime, let me know if you need more help or have feedback

P.S The WDACConfig module will have a GUI soon, in addition to the current CLI experience
